jvm/ml4ir-inference/scripts/generate_model.sh

```diff
@@ -20,9 +20,19 @@ $EXECUTOR ${CLASSIFICATION_MODEL}/pipeline.py \
     --data_dir ${CLASSIFICATION_MODEL}/tests/data/csv/  \
     --feature_config ${CLASSIFICATION_MODEL}/tests/data/configs/feature_config.yaml \
     --model_config ${CLASSIFICATION_MODEL}/tests/data/configs/model_config.yaml \
-    --run_id ${1} \
+    --run_id ${1}_classification \
     --data_format csv \
     --execution_mode train_inference_evaluate \
     --batch_size 32
 
+
+RANKING_MODEL=ml4ir/applications/ranking
```

Comment:
@ducouloa Can you add a section in the readme to explain how this script is being used? I assumed the Java test would call this bash script, but doesn't look like that is happening. Is it the other way around?

Reply:
@lastmansleeping it's used by the `mvn verify -Pintegration_tests` test step - does the README not explain this well enough?  https://github.com/salesforce/ml4ir#continuous-integration

Reply:
No, that works. I think a line or two about what the end to end test is doing specifically would be useful. My intention is to say some DS with minimal JVM knowledge trained a model with ml4ir and wanted to verify if they could deploy it, how would they verify it. We have the code right now, would be nice to have the steps to do that too. If this requires additional work to generalize, we can create a follow up WI.